server/controllers/auth.controller.ts

```diff
@@ -39,6 +39,35 @@ class AuthController {
             return res.formatResponse(e, 'Something bad happened', 400)
         }
     }
+
+    async get(req, res) {
+        try {
```

Comment:
check the absence of `id` at first

Reply:
fix. But authMiddleware checks the validity, which is farmed from id. Checking for id is superfluous.